var/spack/repos/builtin/packages/truchas/package.py

```diff
@@ -70,6 +71,7 @@ class Truchas(CMakePackage):
     # Solvers
     # ------------------------------------------------------------ #
     depends_on("hypre@2.20: ~fortran")
+    depends_on("netlib-lapack")
```

Comment:
```suggestion
    depends_on("netlib-lapack", when="@23:")
```

Looks like the `lapack` dependency is new as of v23 so we probably don't need to make it a dependency of previous versions.

Reply:
This was actually a dependency before too, we just had missed it. I only caught it now because I tried building on a system which didn't already have lapack.

Reply:
Ahh gotcha, I didn't see it listed as a requirement in previous versions of the `BUILDING.md` and figured it was new

Reply:
Oh that's a good catch. In fact I think it's still missing from `BUILDING.md` (even though it's in our `CMakeLists.txt`). I'll have to correct that.

Reply:
I think the latest version of the `BUILDING.md` mentions it just briefly in the requirements. However, a bit more info is always helpful in a quick start guide!

> - LAPACK library or equivalent